client/task-list/tasks/payments/generic-payment-step.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { Form, Link, Stepper, TextControl } from '@woocommerce/components';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+export function GenericPaymentStep( {
```

Comment:
A more generic payment step that can easily be re-used, using this currently for `payfast` and `paystack`, I removed the `payfast.js` one in replacement of this.

I passed in the `methodConfig` as well, the data defined in `methods`. Adding some extra fields like: `fields`, `apiDetailsLink`, `getOptions`

Reply:
Thanks for adding this, this should clean things up going forward.